WordPress/UITestsFoundation/Globals.swift

```diff
@@ -24,6 +24,17 @@ public func navigateBack() {
     }
 }
 
+// Generic function for cases where ScreenObject times out due to long loading time
+// making the Editor Screen evaluate to 'nil'
+public func closeEditorDiscardChanges() {
+    let app = XCUIApplication()
+    let closeButton = app.buttons["Close"]
+    let discardChangesButton = app.buttons["Discard"]
+
+    if closeButton.waitForIsHittable() { closeButton.tap() }
+    if discardChangesButton.waitForIsHittable() { discardChangesButton.tap() }
```

Comment:
Another little nitpick. It's often recommended to keep assignments as close as possible to the code that uses them (spacial locality).

```suggestion
    let closeButton = app.buttons["Close"]
    if closeButton.waitForIsHittable() { closeButton.tap() }
    
    let discardChangesButton = app.buttons["Discard"]
    if discardChangesButton.waitForIsHittable() { discardChangesButton.tap() }
```

The advantage of this style is that one doesn't need to look far to know where `closeButton` or `discardChangesButton` come from when seeing them used in their respective `if` statement.